angular-todo/e2e/virtual-authenticator.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { test, expect } from '@playwright/test';
+
+test('webAuth login angular', async ({ browser, page }) => {
+  let authenticator;
+  if (browser.browserType().name() === 'chromium') {
+    const cdpSession = await page.context().newCDPSession(page);
+
+    await cdpSession.send('WebAuthn.enable');
+    authenticator = await cdpSession.send('WebAuthn.addVirtualAuthenticator', {
+      options: {
+        protocol: 'ctap2',
+        transport: 'internal',
+        hasUserVerification: true,
+        isUserVerified: true,
+        hasResidentKey: true,
+      },
+    });
+    console.log('Authenticator created');
+
+    cdpSession.on('WebAuthn.credentialAdded', () => {
+      console.log('Credential Added');
```

Comment:
would it make more sense to handle expects here in the call back after we confirm the credential was added? Or do we not need this listener at all?

Reply:
not needed, removed. Added to keep track of the status of the flow.